### DISCUSSION TITLE
Adding external compositor like compton to the yocto build

### DISCUSSION THREAD

**Discussion by VishwanathB45** [OE4T/meta-tegra#1358] [Q&A]
Hi,
I'm using kirkstone-l4t-r32.7.x branch in meta-tegra.I want to add compositor to the core-image-sato.how can I add the compositor like compton to image-sato?

**Comment by kekiefer** [comment]
You can see some discussion of wlroots compositors in https://github.com/OE4T/meta-tegra/issues/1209 -- the comments there still stand that nvidia's libdrm is still too old to support wlroots without modification.

On the other hand, Qt-based wayland compositors work well. Qt's eglstreams and gbm backends will both work with oe4t.

**Reply by VishwanathB45** [reply]
@kekiefer Thank you for sharing this information.
It's good to know that Qt-based Wayland compositors seem to have better compatibility with NVIDIA hardware through their support for eglstreams and gbm backends.

**Reply by VishwanathB45** [reply]
@kekiefer 
The reference which you had provided is for weston/wayland.
So I tried using core-image-weston and it is always rebooting/resetting after "Starting: dropbear ssh server" phase.
I'm not sure how to fix this.

As a parallel task, I was planning to use Compton compositor in core-image-sato image. Is it possible to include compton in sato build?

**Reply by VishwanathB45** [reply]
Does anyone got solution to include the Compton compositor in the core-image-sato image?
